docs/developer/permissions.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Permissions
+sidebar_label: Permissions
+---
+
+## User permissions
+
+Its group defines user permissions. Organizing access rights into [Grups](developer/api-reference/objects/group.mdx) helps in determining staff users' roles in the organization.
+
+Examples of groups:
+- Translators - `MANAGE_TRANSLATIONS` permission.
+- Customer support - `MANAGE_ORDERS` and `MANAGE_USERS` permissions.
+
+### Creating and removing groups
+To create a new group, use [permissionGroupCreate](developer/api-reference/mutations/permission-group-create.mdx).
+
+Request:
+```graphql
+mutation {
+  permissionGroupCreate(
+    input: {
+      addPermissions: [MANAGE_GIFT_CARD, MANAGE_DISCOUNTS]
+      addUsers: []
+      name: "Sale managers"
+    }
+  ) {
+    errors {
+      message
+    }
+    group {
+      id
+      name
+      permissions {
+        name
+      }
+    }
+  }
+}
+```
+
+Successful response:
+```json
+{
+  "data": {
+    "permissionGroupCreate": {
+      "errors": [],
+      "group": {
+        "id": "R3JvdXA6NDM=",
+        "name": "Sale managers",
+        "permissions": [
+          {
+            "name": "Manage sales and vouchers.",
+          },
+          {
+            "name": "Manage gift cards.",
+          }
+        ]
+      },
+    }
+  }
+}
+```
+
+To remove group, use [permissionGroupDelete](developer/api-reference/mutations/permission-group-delete.mdx) mutation.
+
+### Modifying group and managing its members
+
+[permissionGroupUpdate](developer/api-reference/mutations/permission-group-update.mdx) takes a list of user IDs you would like to add or remove from the group. Having a user in both of the lists will result in an error.
+
+Example request:
+```graphql
+mutation {
+  permissionGroupUpdate(
+    id: "R3JvdXA6NDM="
+    input: {
+      name: "Sale managers"
+      addPermissions: []
+      removePermissions: []
+      addUsers: ["VXNlcjozMg=="]
+      removeUsers: []
+    }
+  ) {
+    errors {
+      message
+    }
+  }
+}
+```
+
+## App permissions
+
+In contrast to users, app permissions are defined per-app basis. Access can be assigned during app installation and modified later using [appUpdate](developer/api-reference/mutations/app-update.mdx) mutation.
+
+
+## JWT token and permissions
+
+JWT tokens have a list of assigned permissions. Decoding its contents using HS256 algorithm you will get:
```

Comment:
HS256 is a signature algorithm, did you mean to decode the JWT token payload instead?

Reply:
Thats what I meant. Good feedback 👍

Reply:
```suggestion
JWT tokens have a list of assigned permissions. By decoding payload using HS256 algorithm you will get:
```